index.html

```diff
@@ -157,9 +157,46 @@ <h3>Skills</h3>
                     <li>Codepen</li>
                     <li>GitHub</li>
                     <li>GitLab</li>
-                    <li>Terminal/li>
+                    <li>Terminal</li>
                 </ul>
             </div>
         </section>
-        </body>
-        </html>
\ No newline at end of file
+
+        <footer>
+            <section class="footer-section">
+                <h2 class="name_2">Get started</h2>
+                <p class="info_2">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+            
+<form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/{form_id}"
+    method="post">
```

Comment:
- [ ] After submitting your form, I'm getting this error

![image](https://user-images.githubusercontent.com/98453979/199318900-1053bbfd-08be-4849-89da-147bc2ae261c.png)

- [ ] Kindly create a form with [formspree service](https://formspree.io/html) from where a link with a unique `id` will be generated. You will then use that link in your `action` for the form to be submitted correctly

Reply:
Thanks for the feedback.
I'll fix that issue.